src/main/java/seedu/address/logic/commands/fixedexpense/SortFixedExpenseCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands.fixedexpense;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.listmanagers.fixedexpense.FixedExpense;
+
+/**
+ * Sorts your Fixed Expense List according to ascending or descending amount.
+ */
+public class SortFixedExpenseCommand extends Command {
+
+    public static final String COMMAND_WORD = "sortexpense";
+
```

Comment:
Suggestion to use constant keywords or integer for sorting. 
```suggestion
    public static final String SORT_METHOD_HIGH = "high"
    public static final String SORT_METHOD_LOW = "low"
```
This will reduce magic literals in your code.